static/styles/widgets.scss

```diff
@@ -16,42 +16,60 @@
     font-weight: 600;
 }
 
+.todo-widget h4,
 .poll-widget h4 {
-    font-size: 14px;
+    font-size: 18px;
     font-weight: 600;
 }
 
+.todo-widget li,
 .poll-widget li {
     list-style: none;
     margin: 2px 2px 2px 0px;
 }
 
+.todo-widget ul,
 .poll-widget ul {
     margin: 0px 0px 5px 0px;
     padding: 0;
 }
 
-.poll-widget .poll-names {
-    font-size: 10px;
+.poll-widget .poll-names .todo-widget {
+    font-size: 14px;
     color: green;
 }
 
 .poll-widget .poll-option {
     font-weight: 600;
 }
 
-.poll-vote {
+button.task, .poll-vote {
```

Comment:
For multiple selectors like this, our coding convention is to put each on its own line:

~~~ cs
button.task,
.poll-vote {
   ...
}
~~~

The rationale is that it's easy to scan vertically than hortizontally, plus it makes it easier to spot nested styles like `.outer .inner`.